src/idd/idd.utils.js

```diff
@@ -235,6 +235,39 @@ InteractiveDataDisplay.Utils =
             return { minx: minx, maxx: maxx, miny: miny, maxy: maxy };
         },
 
+        enumPlots: function (masterPlot, plots) {
+            plots.push(masterPlot);
+            if (masterPlot.children)
+                masterPlot.children.forEach(function (child) {
+                    InteractiveDataDisplay.Utils.enumPlots(child, plots);
+                });
+            plots.sort(function (a, b) { return b.order - a.order; });
+            return plots;
+        },
+        reorder: function (MP,p, p_before, isPrev) {
```

Comment:
@ElenaPochernina What is MP here? If it is master plot, remove the parameter and use p.master instead.